grails-app/services/org/broadinstitute/orsp/QueryService.groovy

```diff
@@ -1278,6 +1278,18 @@ class QueryService implements Status {
         results
     }
 
+    @SuppressWarnings("GroovyAssignabilityCheck")
+    StorageDocument findAttachmentByUuid(String uuid) {
+        final session = sessionFactory.currentSession
+        final String query = ' select * from storage_document where uuid = :uniqueId '
```

Comment:
`uniqueId` is misleading, that could be anything. Suggest using the actual thing passed in:
```suggestion
        final String query = ' select * from storage_document where uuid = :uuid '
```